src/panels/lovelace/cards/hui-picture-elements-card.ts

```diff
@@ -11,6 +11,10 @@ import { LovelaceElementConfig, LovelaceElement } from "../elements/types";
 interface Config extends LovelaceCardConfig {
   title?: string;
   image: string;
```

Comment:
image can be undefined now right? so `image?: string;`

Reply:
Yes. Good catch. Thanks.